2020/10/2020-10-24-Fuck-the-RIAA.md

```diff
@@ -0,0 +1,102 @@
+October 23, 2020
+
+GitHub
+
+Dear Sir or Madam:
+
+I am contacting you on behalf of the Free Software community and its members.
+
+Fuck the M-P-double-A comin' straight out the underground
+A young pirate got it bad cause I'm down,
+Loadin' DVDs like a motherfuckin' fiend,
+Bring my camera to the movies and I put em on BT,
+Back all that stolen content up on DVD-ROMs,
+'Cuz my tip's been piracy since I dropped out my Mom's,
+And just because I share my MP3s,
+They got the government comin' after me,
+Instead of suin' kids why don't you step on up,
+And release a couple albums that don't completely suck,
+Stop puttin DRM onto audio CDs,
+That don't make it any harder to steal your MP3s,
+If I want your shit for free, I ain't gonna have to pay,
+And all your bullshit is why the fuck I say,
+Hack the Gibson, hack the Gibson,
+I'm seedin' BitTorrents like a digital pimp, son.
+
+Fuck the M-P-double-A
+Fuck the R-I-double-A
+Fuck the suits behind the BSA
+And fuck 'em all for the DMCA
+Fuck the M-P-double-A
+Fuck the R-I-double-A
+Fuck the suits behind the BSA
+And fuck 'em all for the DMCA
+
+Robot pirates, we get our shit for free,
+Parental Advisory you'll never fuckin' see,
+Been a couple years since I seen an FBI warning,
```

Comment:
FBI is nonfree software

Reply:
This is why the FBI warning has been removed. Shit's mad boring.